client/checkout/blocks/upe-fields.js

```diff
@@ -294,6 +294,7 @@ const WCPayUPEFields = ( {
 							paymentIntentSecret,
 							elements,
 							billingData,
+							customerData.shippingAddress,
```

Comment:
That's inconsistent with how billing data are supplied, can we introduce `shippingData`  in the same way how `billingData` is declared?

Reply:
Thanks for spotting this. I have fixed this here - 514fa82 
I hope its correctly done. Manual test seems fine.